forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/PlatformUsernamePasswordNodeListener.java

```diff
@@ -7,14 +7,14 @@
 
 package org.forgerock.android.auth;
 
+import static org.forgerock.android.auth.AndroidBaseTest.PASSWORD;
```

Comment:
Revert change or update copyright

Reply:
Reverted the changes and left the original copyright note...